src/test/java/io/jenkins/plugins/analysis/warnings/recorder/pageobj/DuplicationTable.java

```diff
@@ -65,10 +65,16 @@ public DuplicationTable(final HtmlPage page, final boolean hasPackages) {
                     "Details", "File", "Severity", "#Lines", "Duplicated In", "Age");
         }
 
+        while (table.getBodies().isEmpty()) {
+            System.out.println("Waiting for table to be initialized ...");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/app/uhafner/warnings-ng-plugin/pullRequest?prid=4663559)